bot/controllers/twitchLive.js

```diff
@@ -11,16 +11,15 @@ class StreamerLiveController extends BaseController {
         '!twitchLive',
         '!twitchLive <twitch_name>',
         'Twitch Live',
-        'A quick easy way to ',
+        'A quick easy way to announce you are going live on Twitch.',
```

Comment:
Grammar suggestion:

'A quick easy way...'
vs.
'A quick and easy way...'

Reply:
fixed the grammar issue.